MovieApp/MainActivity.cs

```diff
@@ -0,0 +1,516 @@
+
+
+
+            int contador = 0;
+
+            //foreach (var i in mov.results)
+            //{
+            //    string genero = "";
+            //    for (int g = 0; g < i.genre_ids.Count; g++)
+            //    {
+            //        if (genero.Length > 0)
+            //        {
+            //            genero += "/";
+            //        }
+            //        var itemGenero = listaGenero.Where(x => x.idGenero == i.genre_ids[g]).ToList()[0];
+            //        genero += itemGenero.genero;
+            //    }
+
+            //    //List of Details                
+            //    string endereoDetail = "https://api.themoviedb.org/3/movie/" + i.id + "?api_key=" + apiKey + "&language=en-US";
+            //    string responseFromServerImage = CallRequest(endereoDetail);
+            //    var detail = JsonConvert.DeserializeObject<RootObjectDetail>(responseFromServerImage);
+            //    string caminhoImagem = detail.poster_path;
+
+            //    items[contador] = i.title + " - " + caminhoImagem + " - " + genero + " - " + i.release_date;
+            //    contador++;
+
+            //}
+
+
+
+            //for (int i = 1; i < mov.total_pages - 1; i++)
+            contador = 0;
+            items.Add("Title" + "   ImagePath" + "    Genre" + "     Release Date");
+            for (int i = 1; i < 4; i++)
+            {
+                pagina = i.ToString();
+                endereco = "https://api.themoviedb.org/3/movie/upcoming?api_key=" + apiKey + "&language=en-US&page=" + pagina;
+                string retorno = CallRequest(endereco);
+
+                var movies = JsonConvert.DeserializeObject<Movie>(retorno);
+                foreach (var m in movies.results)
+                {
+
+                    try
+                    {
+                        contador++;
+
+                        string genero = "";
+                        for (int g = 0; g < m.genre_ids.Count; g++)
+                        {
+                            if (genero.Length > 0)
+                            {
+                                genero += "/";
+                            }
+                            var itemGenero = listaGenero.Where(x => x.idGenero == m.genre_ids[g]).ToList()[0];
+                            genero += itemGenero.genero;
+                        }
+
+                        //List of Details                
+                        string endereoDetail = "https://api.themoviedb.org/3/movie/" + m.id + "?api_key=" + apiKey + "&language=en-US";
+                        string responseFromServerImage = CallRequest(endereoDetail);
+                        var detail = JsonConvert.DeserializeObject<RootObjectDetail>(responseFromServerImage);
+                        //string caminhoImagem = detail.poster_path;
+                        string caminhoImagem = "?";
+
+                        items.Add(detail.title + " - " + caminhoImagem + " - " + genero + " - " + detail.release_date);

```

Comment:
this and all strings concatenations you did could be written as:
items.Add($"{detail.title} - ? - {genero} - {detail.release_date}");